### PR TITLE
Added add contact form

### PR DESCRIPTION
**Pull request by raeeschachar**

### CODE REVIEW

**Comment by raeeschachar** [comment]
@MaryKhan

File: contacts/forms.py [hunk 1]
```diff
@@ -0,0 +1,9 @@
+from django import forms
+
+from contacts.models import Contact
+
+
+class AddContactForm(forms.ModelForm):
```
Comment (MaryKhan) [inline contacts/forms.py]: seems like we're not using this form anywhere.

**Review by MaryKhan — CHANGES_REQUESTED** [review]

File: contacts/templates/contacts/contact_list.html [hunk 2]
```diff
@@ -6,5 +6,7 @@ <h1>My Contacts</h1>
     {% for contact in my_contacts %}
         <li><a href="{% url 'contacts:detail' contact.id %}">{{ contact }}</a></li>
     {% endfor %}
+    
+    <a href="/contacts/add_contact">Add Contact</a>
```
Comment (MaryKhan) [inline contacts/templates/contacts/contact_list.html]: use `{% url '<url-name>' %}` instead.

File: contacts/views.py [hunk 3]
```diff
@@ -12,6 +12,8 @@ class ContactDetailView(generic.DetailView):
     model = Contact
 
 
-class AddContactView(generic.CreateView):
+class AddContactView(CreateView):
     model = Contact
-    fields = '__all__'
+    fields = ['name', 'email', 'phone_number']
+
+    success_url = "/contacts"
```
Comment (MaryKhan) [inline contacts/views.py]: same here!
don't use hard-coded url anywhere in project.

**Review by MaryKhan — CHANGES_REQUESTED** [review]

**Pull request merged**